Dockerfile

```diff
@@ -1,16 +1,7 @@
-FROM node:carbon
-
-# Create app directory
-RUN mkdir -p /app
+FROM node:current
 WORKDIR /app
-
-# Bundle app source
-COPY . /app
+COPY package.json /app
 RUN npm install
-
-# Mount persistent storage
-VOLUME /app/data
-VOLUME /app/public/uploads
-
+COPY . /app
```

Comment:
Wondering if this would potentially overwrite what gets installed in the above `RUN` if the host machine has already `npm install`ed.